examples/platforms/efr32mg21/efr32mg21.ld

```diff
@@ -0,0 +1,242 @@
+/***************************************************************************//**
+ * @file
+ * Linker script for Silicon Labs EFR32MG21 devices
+ * @version 5.8.1
+ *******************************************************************************
+ * # License
+ * <b>Copyright 2019 Silicon Laboratories, Inc. www.silabs.com </b>
```

Comment:
2020

Reply:
File was created in 2019 so this is correct.